tests/causal_refuters/test_add_unobserved_common_cause.py

```diff
@@ -100,8 +101,109 @@ def test_linear_sensitivity_analysis(self, mock_fig,estimator_method,
         refute = model.refute_estimate(target_estimand, estimate ,
                                method_name = "add_unobserved_common_cause",
                                simulated_method_name = simulated_method_name, 
-                               benchmark_covariates = benchmark_covariates,
+                               benchmark_common_causes = benchmark_common_causes,
                                effect_fraction_on_treatment = effect_strength_on_t)
+        
+        if refute.confounder_increases_estimate == True:
+                bias_adjusted_estimate = refute.benchmarking_results['bias_adjusted_estimate']
+                assert all(estimate <= refute.estimate for estimate in bias_adjusted_estimate) #if confounder_increases_estimate is True, adjusted estimate should be lower than original estimate
+        else:
+                bias_adjusted_estimate = refute.benchmarking_results['bias_adjusted_estimate']
+                assert all(estimate >= refute.estimate for estimate in bias_adjusted_estimate)
+
+        #check if all partial R^2 values are between 0 and 1
+        assert all((val >= 0 and val <=1) for val in refute.benchmarking_results['r2tu_w']) 
+        assert all((val >= 0 and val <=1) for val in refute.benchmarking_results['r2yu_tw'])
+        assert  refute.stats['r2yt_w'] >= 0 and refute.stats['r2yt_w'] <= 1
+
         assert refute.stats['robustness_value'] >= 0 and refute.stats['robustness_value'] <= 1
         assert mock_fig.call_count > 0  # we patched figure plotting call to avoid drawing plots during tests
 
+    @pytest.mark.parametrize(["estimator_method", "effect_strength_on_t", "benchmark_common_causes", "simulated_method_name"],
+                             [("backdoor.linear_regression", [1,2,3], ["W3"], "PartialR2"),])
+    @patch("matplotlib.pyplot.figure")
+    def test_linear_sensitivity_analysis2(self, mock_fig,estimator_method,
```

Comment:
can you give descriptive function names rather than 1, 2, 3?
e.g. test_linear_sensitivity_strength_of_confounding for 2
And test_linear_sensitivity_output_range for the 3 test